content/docs/3.0/tasks/deploy/binary.md

```diff
@@ -9,13 +9,16 @@ menu:
 
 This guide describes how to deploy a TiKV cluster using binary files.
 
-{{< warning >}}
-**Warning:** Do not use binaries to deploy the TiKV cluster in the production environment. For production, [use Ansible to deploy the TiKV cluster](../ansible).
-{{< /warning >}}
 
 - To quickly understand and try TiKV, see [Deploy the TiKV cluster on a single machine](#deploy-the-tikv-cluster-on-a-single-machine).
 - To try TiKV out and explore the features, see [Deploy the TiKV cluster on multiple nodes for testing](#deploy-the-tikv-cluster-on-multiple-nodes-for-testing).
 
+{{< warning >}}
+The TiKV team strongly recommends you use the [**Ansible Deployment**](../ansible/) method. It is the method our team uses when we assist with deployment.
```

Comment:
Maybe we should join these "It is the method our team uses when we assist with deployment." and "Other methods are documented for informational purposes." in one line. This makes a clearer context for  "other methods"

Reply:
Also, this does not seem like a Warning given the information it presents, maybe a Note? (Warning is more about preventing users from something that may cause consequences. This info here gives recommendations)

Reply:
In this case, choosing a different deployment method will likely have quite a few consequences, so I think a warning is appropriate. :) Notes may be glossed over.